### ISSUE TITLE
The `show` command should warn, not fail, if the plan is stale.

### ISSUE THREAD

**Issue by nl-brett-stime** [hashicorp/terraform#30269]
Sometimes it's useful to see what the changes related to a give plan *were* even after it's been applied. When using the `show` command on a stale plan, ideally Terraform would report a warning and continue to show the plan instead of reporting an error and terminating. Alternately, maybe add an option to ignore staleness. E.g., `terraform show -ignore-stale -json some.tfplan`

### Terraform Version

Tested in `v1.0.11` but probably applies to more recent versions as well.

### Terraform Configuration Files

Any config

### Expected Behavior
```
<shows plan or plan JSON>
<outputs *warning* in yellow, maybe to stderr, that plan is stale>
```

### Actual Behavior
```
(red error message): Saved plan is stale
<program terminates>
```

### Steps to Reproduce

1. `terraform init`
2. `terraform plan -out some.tfplan`
3. `terraform apply some.tfplan`
4. `terraform show -json some.tfplan`

**Comment by alisdair** [comment]
Hi @nl-brett-stime, thanks for submitting this. There's some related work happening in #30205, fixing #30195, and I believe that what you're describing here may already be fixed in the 1.1 series. Could you check if 1.1.2 resolves the issue for you?

**Comment by dominics** [comment]
Shouldn't that be `1.1.3` to check> (if we're checking for #30205 - the changelog indicates that's when it was released)

#28136 seems like another duplicate that might be fixed

**Comment by alisdair** [comment]
Ah, yes, thanks for the correction!

I've just verified that this is fixed in Terraform 1.1.3, so I'm closing this and the other issue accordinly.
